Makefile

```diff
@@ -149,7 +159,8 @@ $(IMAGE_DIRS): %: %/Dockerfile check-links
 	@echo "Building [$*] image"
 	@echo
 	cd $(dir $<) && time $(SHELL) -c "( tar -czh . | docker build $(DBFLAGS_$@) -t $@ --label $(LABEL) - )"
-	@echo
+
+$(IMAGE_TESTS): test-%: % %/capabilities.txt
```

Comment:
Maybe make release should still run the tests, separating out the image building is good though so you can test that separately.